resilience4j-spring/src/test/java/io/github/resilience4j/fallback/FallbackMethodTest.java

```diff
@@ -56,6 +56,16 @@ public void fallbackGlobalExceptionWithSameMethodReturnType() throws Throwable {
             .isEqualTo("recovered-IllegalStateException");
     }
 
+    @Test
+    public void fallbackGlobalExceptionWithSameMethodReturnTypeAndMultipleParameters() throws Throwable {
+        FallbackMethodTest target = new FallbackMethodTest();
+        Method testMethod = target.getClass().getMethod("multipleParameterTestMethod", String.class, String.class);
+        FallbackMethod fallbackMethod = FallbackMethod
+            .create("fallbackMethod", testMethod, new Object[]{"test", "test"}, target);
+        assertThat(fallbackMethod.fallback(new IllegalStateException("err")))
```

Comment:
just cosmetic/convention changes (empty lines):

```
FallbackMethod fallbackMethod = FallbackMethod
            .create("fallbackMethod", testMethod, new Object[]{"test", "test"}, target);

String fallback = fallbackMethod.fallback(new IllegalStateException("err");

 assertThat(fallback)).isEqualTo("recovered-IllegalStateException");
```

Reply:
Added the empty lines. Also I had it tested with the previous implementation to see that the test fails 🙂